doc/usage.md

```diff
@@ -166,6 +166,138 @@ $ oc get nodes -w
 Once the nodes reboot, you might want to run another Suite to ensure that
 the remediation that you applied previously was no longer found.
 
+## Evaluating rules against default configuration values
+
+Kubernetes infrastructure may contain incomplete configuration files. At run time, 
+nodes will assume default configuration values for missing configuration options.
+And some configuration can be passed as command line arguments, therefore,
+the Compliance Operator cannot assume the configuration file on the node is complete
+and it may be missing options used in ru le checks.
+
+To prevent false negative findings where the default configuration value passes
+a check, the Compliance Operator generates a complete set of configuration
+values, used specifically for evaluating rules. This increases the accuracy
+of the scan results.
+
+### How to use it?
+
+#### Standard cluster with only `master` and `worker` pool
+In a standard cluster that only has `master` and `worker` pool, there isn't much needed
+to be done, everything will work out of the box.
```

Comment:
I'd consider adding an additional statement to the bottom of the previous section, and removing this section.

My thought for suggesting this is that we have nice introduction of the feature above, and we wrap that up by saying "Oh, and this works by default. You don't need to do anything if you use the default node pools. Otherwise, keep reading."

For example:

```
To prevent false negative findings where the default configuration value passes
a check, the Compliance Operator generates a complete set of configuration
values, used specifically for evaluating rules. This increases the accuracy
of the scan results.

No additional changes are required to use this feature with `master` and
`worker` node pools. See the following sections for details on how to
use this feature with custom node pools.

### Custom node pools

For scalability reasons, the Compliance Operator doesn't persist a copy of
each node configuration. Instead, it aggregates consistent configuration
options for all nodes within a single node pool into one copy of the configuration
file. It then uses the configuration file for a particular node pool to evaluate
rules against nodes within that pool.

If your cluster uses custom node pools outside the default `worker` and `master`
node pools, you'll need to supply additional variables to ensure the Compliance
Operator aggregates a configuration file for that node pool.

# pick up at line 190?
...
```

Reply:
that's a really good idea, thanks for it